setup.py

```diff
@@ -13,7 +13,8 @@
 ]
 
 extras_require = {
-    ':python_version < "3"': 'py2-ipaddress >= 3.4.1',
+    ':python_version < "3.5"': 'backports.ssl_match_hostname >= 3.5',
+    ':python_version < "3.5"': 'ipaddress',
```

Comment:
`ipaddress` is available from python 3.3 onwards. Change in #1039 is what I meant and seems to be working fine.